src/components/PlaylistManager/Panel/index.js

```diff
@@ -30,6 +30,9 @@ const makeActions = ({
     {(index > 0 || isFiltered) && (
       <MoveToFirstAction onFirst={() => onMoveToFirst(media, selection)} />
     )}
+    {((index > 0 && index !== (selection.items.length - 1)) || isFiltered) && (
+      <MoveToLastAction onLast={() => onMoveToLast(media, selection)} />
+    )}
     {(index === 0 && !isFiltered) && (
     <MoveToLastAction onLast={() => onMoveToLast(media, selection)} />
     )}
```

Comment:
Could we combine these conditions that both render a `<MoveToLastAction/>`? 🤔 
Maybe just this could work:

```suggestion
    {(index !== (selection.items.length - 1) || isFiltered) && (
      <MoveToLastAction onLast={() => onMoveToLast(media, selection)} />
    )}
```

In an unfiltered list, we show it if the item is not in last position. In a filtered list, we don't know what the item's _actual_ position in the playlist is, so we always show it just in case.

Reply:
Yeah, looks pretty straightforward.